frontend/lib/src/setupTestEnv.js

```diff
@@ -33,3 +34,14 @@ console.error = (...args) => {
   // For all other warnings, call the original console.warn
   originalConsoleError(...args)
 }
+
+// apache-arrow requires these to be defined
```

Comment:
**note:** I seem to be getting an error when running tests locally even in `develop`. It seems to have started with https://github.com/streamlit/streamlit/commit/29bf74fb099dc071bcdc319f6657bea89db009a6 . This should fix it, although it isn't a result of the changes in this PR directly.